Src/Plugins/Tests.Security/Validators/SEC101_006.GitHubPatValidatorTests.cs

```diff
@@ -24,7 +24,9 @@ public void GitHubPatValidator_TestStatic()
             groups.Add("checksum", new FlexMatch() { Value = "checksum" });
             groups.Add("scanTargetFullPath", new FlexMatch() { Value = "GitHitPatTest" });
 
-            IEnumerable<ValidationResult> validationResults = GitHubPatValidator.IsValidStatic(groups);
+            var gitHubPatValidator = new GitHubPatValidator();
```

Comment:
>var [](http://example.com/codeflow?start=12&length=3)

in other rules, you don't introduce a vertical line after instantiating the rule. :) Style nit!